app/controllers/responsibilities_controller.rb

```diff
@@ -21,7 +21,7 @@ def confirm
   def create
     @responsibility = Responsibility.create! responsibility_params
 
-    me = PrisonOffenderManagerService.get_signed_in_pom_details(current_user, @prison).emails.try(:first)
+    me = Nomis::Elite2::UserApi.user_details(current_user).email_address.first
```

Comment:
Still worth wrapping the email call with try() in case it’s nil

Reply:
Good idea! Done and done